packages/vendure-plugin-invoices/src/api/invoice.service.ts

```diff
@@ -198,6 +201,11 @@ export class InvoiceService implements OnModuleInit, OnApplicationBootstrap {
       orientation: 'portrait',
       border: '10mm',
       timeout: 1000 * 60 * 5, // 5 min
+      childProcessOptions: {
+        env: {
+          OPENSSL_CONF: '/dev/null',
+        },
+      },
```

Comment:
Nice one! I think this means we don't need this OPENSSL_CONF in the env anymore? If so, can you also remove it from the Readme.